src/internal/m365/collection/site/collection.go

```diff
@@ -355,8 +356,17 @@ func (sc *Collection) handleListItems(
 	metrics.Bytes += size
 	metrics.Successes++
 
+	template := ""
+	if list != nil &&
+		list.GetList() != nil {
```

Comment:
nit: I think this is short enough to inline.
```suggestion
	if list != nil && list.GetList() != nil {
```